ftw/simplelayout/migration.py

```diff
@@ -96,15 +103,61 @@ def __call__(self):
 )
 
 
-
-def move_sl_block_into_slot(page, block, slot_name):
-    page_configuration = IPageConfiguration(page)
+def move_sl_block_into_slot(old_page, new_page, block, slot_name):
+    page_configuration = IPageConfiguration(new_page)
     page_state = page_configuration.load()
+
+    # initiate layout if it is in its initial state
+    if slot_name == 'default' and page_state == page_configuration._default_page_config():
+
+        # setup different layouts
+        if slot_name == 'default' and ISimplelayoutTwoColumnView.providedBy(old_page):
+            # two columns
+            page_state[slot_name] = [
+                {'cols': [ {'blocks': []}, {'blocks': []} ]}
+            ]
+
+        elif slot_name == 'default' and ISimplelayoutTwoColumnOneOnTopView.providedBy(old_page):
```

Comment:
The condition `slot_name == 'default'` is always `True` at this point and could be removed for simplification.